src/vkscript/command_parser.cc

```diff
@@ -307,6 +309,74 @@ Result CommandParser::ProcessDrawRect() {
   return {};
 }
 
+Result CommandParser::ProcessDrawGrid() {
```

Comment:
Drop this part. We aren't updating the vkscript format with new features. If you want to use `draw grid` then AmberScript is the way to go. We also don't want to diverge from the original vkscript as much as possible.

Reply:
Removed all changes to vkscript.